source/roadmap.html.erb

```diff
@@ -43,40 +43,53 @@ title: Roadmap
 
       <h2 class="heading-large">Roadmap</h2>
       <p>This roadmap is for informational purposes only and may be affected by changing priorities. You shouldn’t rely on this information for major purchasing or planning purposes.</p>
-      
+
       <h3 class="heading-medium">Near-term</h3>
 
       <ul class="list list-bullet">
+      <li>Support for cloud-based CI tools such as Travis</li>
 		  <li>Redis as a backing service</li>
 		  <li>Self-service SSL certificate management</li>
 	</ul>
 
 	  <h3 class="heading-medium">Medium-term</h3>
 
       <ul class="list list-bullet">
-		  <li>Elasticsearch as a backing service</li>
 		  <li>MongoDB as a backing service</li>
 		  <li>Self-service user management</li>
 		  <li>Enhanced backing service monitoring</li>
+      <li>Elasticsearch as a backing service</li>
```

Comment:
I know that it's minor thing but there is something wrong with indentation here and line 69.